wordpress-mobile/WordPress-FluxC-Android
# ThemeStore: Add actions to delete a theme installed on a Jetpack site

Note: #569 should be merged first

To test:
* Go to the Jetpack site's wp-admin and check which themes are currently installed
* In the example app...sign into an account with the Jetpack site linked
* Go to the Theme's screen
* Type a theme slug (ie `edin`, `twentysixteen`, etc...) for a theme that isn't already installed into the EditText
* Click the `Delete theme on Jetpack site` button and check the log output to make sure there's an error (unknown theme)
* Now try again with an installed theme and make sure the log shows success

cc @aforcier, this one is not very urgent since WPAndroid doesn't expose this to the user yet.
